helm/hpcc/templates/_helpers.tpl

```diff
@@ -239,22 +237,20 @@ Pass in root and includeLabels (optional)
   {{- end }}
  {{- end }}
 {{- end -}}
+{{- /*Backwards compatibility: when no data planes defined in values, create default pvc*/ -}}
+{{- if and (or (has "data" $includeLabels) (has "" $includeLabels)) (not $dataStorage.plane) }}
```

Comment:
similar to comment in addVolumesMount
shouldn't have "data" specific code in this generalized function, should move to addDataVolume
and "" should already be handled, this is going to generate this when not needed afaics (should only generate if datasStorage.plane is not set)

Reply:
I've commented in the code why this is necessary.